fixtures/parallelizer/__init__.py

```
@@ -355,11 +373,18 @@ def _start_slave(self, slaveid):
 
         # worker output redirected to null; useful info comes via messages and logs
         slave = subprocess.Popen(
-            ['python', remote.__file__, slaveid, base_url], stdout=devnull, stderr=devnull)
```

it hides errors at early startup of nodes